cmd/cluster-agent/api/listener.go

```diff
@@ -19,5 +19,5 @@ import (
 
 // getListener returns a listening connection
 func getListener() (net.Listener, error) {
-	return net.Listen("tcp", fmt.Sprintf("0.0.0.0:%v", config.Datadog.GetInt("cmd_port")))
```

Comment:
As the DCA will now only be distributed as a docker image, we don't have to handle both agents running on the same IP.
Can we remove the `clusteragent_cmd_port` and `clusteragent_expvar_port`, and use the same options and defaults as the node agent?

Reply:
Let's keep `clusteragent_cmd_port` for the API the node-agents will use, but the management interface should be listening on localhost on `cmd_port`

Reply:
makes sense!